Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/DocumentPropertyEditor/OverridePropertyHandler.cpp

```diff
@@ -10,24 +10,56 @@
 
 namespace AzToolsFramework::Prefab
 {
-    OverridePropertyHandler::OverridePropertyHandler()
+    PrefabOverrideLabelHandler::PrefabOverrideLabelHandler()
+        : m_overridden(false)
+        , m_iconButton(new QToolButton())
+        , m_textLabel(new AzQtComponents::ElidingLabel())
```

Comment:
uber nit : since you are modifying these within the same constructor, I recommend moving these within the constructor body.